app/org/sagebionetworks/bridge/services/StudyService.java

```diff
@@ -166,6 +176,40 @@ public Study getStudy(StudyIdentifier studyId) {
         return studyDao.getStudies();
     }
 
+    public Study createStudyAndUser(StudyAndUserHolder studyAndUserHolder) throws SynapseException {
+        checkNotNull(studyAndUserHolder, Validate.CANNOT_BE_NULL, "study and users");
+        if (studyAndUserHolder.getUsers() == null) {
+            throw new BadRequestException("User list cannot be null.");
+        }
+        if (studyAndUserHolder.getStudy() == null) {
+            throw new BadRequestException("Study cannot be null.");
+        }
```

Comment:
Is it worth creating a validator for StudyAndUserHolder? This allows us to check for missing (or empty) user list and missing study, as well as validate the parameters in the study and users. This means callers won't have to keep going back and fixing their errors one by one. Rather, they get a big list of all the errors that they can fix at once.